src/popup.cpp

```diff
@@ -62,9 +62,9 @@ void Popup::draw(NVGcontext* ctx) {
     nvgBeginPath(ctx);
     nvgRoundedRect(ctx, mPos.x(), mPos.y(), mSize.x(), mSize.y(), cr);
 
-    nvgMoveTo(ctx, mPos.x()-15,mPos.y()+mAnchorHeight);
```

Comment:
This is drawing the little triangle AFAIK. Why is usage of mAnchorHeight suspect here?